src/main/java/com/game/Battleship/controllers/ShipController.java

```diff
@@ -0,0 +1,30 @@
+package com.game.Battleship.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.game.Battleship.models.Ship;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@RestController
+public class ShipController {
+
+    @GetMapping("/ships")
+    public List<Ship> getShips() {
+
+        HashMap<String, Integer> fleet = new HashMap<String, Integer>();
+        fleet.put("Carrier", 5);
+        fleet.put("Battleship", 4);
+        fleet.put("Cruiser", 3);
+        fleet.put("Submarine",3);
+        fleet.put("Destroyer", 2);
+
+        List<Ship> allShips = new ArrayList<>();
+        fleet.forEach((name, size) -> allShips.add(new Ship(name, size)));
```

Comment:
It seems a bit strange that you're creating a `HashMap` just to iterate through it to create a list of all the ships. Can you initialize each ship without using a `HashMap`?

Reply:
Done